frontend/packages/kubevirt-plugin/src/yaml-templates.ts

```diff
@@ -39,4 +40,77 @@ spec:
           cloudInitNoCloud:
             userDataBase64: SGkuXG4=
 `,
-);
+  )
+  .setIn(
+    [VmTemplateModel, 'default'],
```

Comment:
This will override the default template YAML, which we don't want.

Reply:
Yes, I know ... see https://github.com/openshift/console/pull/1673#issuecomment-503483627 and https://github.com/openshift/console/pull/1673#discussion_r292883105

@spadgett Do we have other option to set specialized yaml if path is `.../vmtemplates/~new` ?

:+1: also will address in #1673 :-)

Reply:
> This will override the default template YAML, which we don't want.

@spadgett @yaacov FYI, we already have an extension check to cover this:

`packages/console-app/src/__tests__/extension-checks/yaml-templates.spec.ts`

So even if a plugin tries to override an existing YAML template (modelRef & templateName composite key), `yarn test` still fails due to `only one named template per model is allowed`.